.github/workflows/activate-docker-distribution.yaml

```diff
@@ -64,18 +64,21 @@ jobs:
           if [[ $EXIT_CODE -eq 0 ]]; then
             echo "ERROR: image ${{ env.DESTINATION_IMAGE_TAG }} already exist"
             exit 1
-          elif [[ $EXIT_CODE -ne 1 ]]
+          elif [[ $EXIT_CODE -ne 1 ]]; then
             echo "ERROR: $EXIT_CODE"
             exit 1
           fi
 
       - name: "Update image tag"
         run: |
           set +e
+          docker pull ${{ env.SOURCE_IMAGE_TAG }}
           docker image tag ${{ env.SOURCE_IMAGE_TAG }} ${{ env.DESTINATION_IMAGE_TAG }}
-          if [[ "${{ inputs.set_latest }}" ]]; then
+          if [[ ${{ inputs.set_latest }} ]]; then
             docker image tag ${{ env.SOURCE_IMAGE_TAG }} latest
           fi
```

Comment:
Is the `docker buildx imagetools`  not available? That would save us from having to pull the image.